src/mongo/installer/msi/wxs/UIFragment.wxs

```diff
@@ -2,11 +2,44 @@
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"  xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
     <Fragment>
         <UI Id="MongoWixUI">
-            <!-- Base our MSI on the FeatureTree based Wix set, and add our custom dialog boxes -->
-            <UIRef Id="WixUI_FeatureTree" />
+            <!-- Based on WixUI_FeatureTree, with our custom dialog boxes -->
+          <UIRef Id="WixUI_Common" />

```

Comment:
- What are the consequences of changing from `WixUI_FeatureTree` to `WixUI_Common`?
- The comment seems out of date, since it still mentions `WixUIFeatureTree`.